frontend/src/components/AddClassModal.tsx

```diff
@@ -260,40 +268,21 @@ export class AddClassModal extends React.Component<
             <XButton onClick={this.prepareToClose.bind(this)}></XButton>
           </CloseButtonWrapper>
           <h1 id="simple-modal-title">Add classes</h1>
-          <form>
-            <FormRow>
-              <StyledLabel>
-                Subject:
-                <input
-                  type="text"
-                  placeholder="CS"
-                  value={this.state.formSubject}
-                  onChange={this.handleSubjectChange.bind(this)}
-                ></input>
-              </StyledLabel>
-              <StyledLabel>
-                Class ID:
-                <input
-                  type="text"
-                  placeholder="3500"
-                  value={this.state.formClassId}
-                  onChange={this.handleClassIdChange.bind(this)}
-                ></input>
-              </StyledLabel>
-            </FormRow>
-          </form>
-          <SearchButton onClick={this.handleSearch.bind(this)}>
-            Search
-          </SearchButton>
+          <FormColumn>
+            {this.renderSubjectTextField()}
+            {this.renderClassIDTextField()}
+          </FormColumn>
+          <NextButton text="Search" onClick={this.handleSearch.bind(this)} />
           {this.renderQueuedClasses()}
-          <SubmitButton
+          <NextButton
+            text={
+              this.state.queuedCourses.length === 1
+                ? "Add Class"
+                : "Add Classes"
+            }
             onClick={this.handleSubmit.bind(this)}
             disabled={this.state.queuedCourses.length === 0}
-          >
-            {this.state.queuedCourses.length === 1
-              ? "Add Class"
-              : "Add Classes"}
-          </SubmitButton>
```

Comment:
Not a huge deal but it feels like there's a lot of space above/beneath the Search button but almost no space under the Add Class button, so if it's not too difficult to balance out the space with those buttons that would be great